### PR TITLE
Add R packages r-callr, r-covr, r-flux, and r-ggsci

### PR DESCRIPTION
**Pull request by jdblischak**
This PR adds the following R packages:

* [r-callr](https://cran.r-project.org/web/packages/callr/index.html)
* [r-covr](https://cran.r-project.org/web/packages/covr/index.html)
* [r-flux](https://cran.r-project.org/web/packages/flux/index.html)
* [r-ggsci](https://cran.r-project.org/web/packages/ggsci/index.html)

I followed the instructions from #2009. I used the script in [conda_r_skeleton_helper
](https://github.com/bgruening/conda_r_skeleton_helper/tree/80c04dd2b8b15fe9a1f070fa2a78a6a4113dfde7) and added myself as a maintainer.

### CODE REVIEW

**Comment by jdblischak** [comment]
Well that didn't go well. I was able to build all 4 of these packages on my local linux-64 machine using `conda build`, but the linter and all 3 CI services failed to parse the YAML files. The error message from the Windows build on AppVeyor is the most succinct:

```
conda build-all recipes --matrix-conditions "numpy >=1.11" "%PY_CONDITION%" "r-base >=3.3.2"
Fetching package metadata .............
----------------------------------------------------------------------
Unable to parse meta.yaml file
Error Message:
--> mapping values are not allowed in this context
-->   in "<unicode string>", line 19, column 7
```

Line 19 (`skip: true  # [win32]`) is causing the YAML parsing problem because its indentation is wrong. The indentation problems were unexpectedly caused by a bug in one of the `sed` commands of `run.sh`. I had subsequently fixed the bug afterwards ([PR](https://github.com/bgruening/conda_r_skeleton_helper/pull/3)) so that it would have its desired effect (which was to remove the string `" | file LICENSE"`, which I had done manually), but I didn't re-run it. I'll fix the indentation.

File: recipes/r-flux/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - r-catools
+
+test:
+  commands:
+    - $R -e "library('flux')"  # [not win]
+    - "\"%R%\" -e \"library('flux')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=flux
+  license: GPL-2
```
Comment (bgruening) [inline recipes/r-flux/meta.yaml]: please add the license file

File: recipes/r-ggsci/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - r-scales
+
+test:
+  commands:
+    - $R -e "library('ggsci')"  # [not win]
+    - "\"%R%\" -e \"library('ggsci')\""  # [win]
+
+about:
+  home: https://ggsci.net, https://github.com/road2stat/ggsci
+  license: GPL-3
```
Comment (bgruening) [inline recipes/r-ggsci/meta.yaml]: please add license file

**Comment by jdblischak** [comment]
Thanks for the review, @bgruening. As requested, I added a license file for the GPL'd packages r-flux and r-ggsci. I followed the example of the recent PR #3202 for r-bookdown.

Also, 6 days after my I submitted my PR, version 3.0.0 of [covr](https://cran.r-project.org/package=covr) was released to CRAN, and the [license was changed from MIT to GPL3](https://github.com/jimhester/covr/issues/256). Thus I updated the recipe and added a license file for it as well.

**Pull request merged**